MAUI/TextInputLayout/States-And-Colors.md

```diff
@@ -0,0 +1,248 @@
+
+Resources = new ResourceDictionary() { style };
+
+inputLayout.Style = style;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![States](images/StatesAndColors/States.png)
+
+## Disabled state
+
+The text input layout is disabled by setting the `IsEnabled` property to `false`. The color of the container and other UI elements will also be changed to the disabled state, but its color cannot be customized.
+
+{% tabs %} 
+
+{% highlight xaml %} 
+
+<inputLayout:SfTextInputLayout Hint="Name" 
+                               IsEnabled="false">
+    <Entry />
+</inputLayout:SfTextInputLayout>  
+ 
+{% endhighlight %}
+
+{% highlight C# %} 
+
+var inputLayout = new SfTextInputLayout();
+inputLayout.Hint = "Name";
+inputLayout.IsEnabled = false;
+inputLayout.Content = new Entry(); 
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Disabled state](images/StatesAndColors/Disabled.jpg)
+
+
+## Container color
+The color of the container can be customized by setting the `ContainerBackgroundColor` property. It is applicable when the `ContainerType` property is set to `Filled` and `Outlined`.
+
+### Filled
+
+The color of the container is customized when the `ContainerType` is `Filled`.
+
+{% tabs %} 
+
+{% highlight xaml %} 
+
+<inputLayout:SfTextInputLayout Hint="Name" 
+                               Stroke="#0450C2"
+                               ContainerType="Filled"
+                               ContainerBackgroundColor="#E6EEF9">
+    <Entry />
+</inputLayout:SfTextInputLayout>  
+ 
+{% endhighlight %}
+
+{% highlight C# %} 
+
+var inputLayout = new SfTextInputLayout();
+inputLayout.Hint = "Name";
+inputLayout.Stroke = Color.FromHex("#0450C2");
+inputLayout.ContainerBackgroundColor = Color.FromHex("#E6EEF9");
+inputLayout.ContainerType = ContainerType.Filled;
+inputLayout.Content = new Entry() { Text = "John" }; 
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Filled](images/StatesAndColors/FilledContainerBackground.png)
+
+### Outlined
+
+The color of the container is customized when the `ContainerType` is `Outlined`.
+
+{% tabs %} 
+
+{% highlight xaml %} 
+
+<inputLayout:SfTextInputLayout Hint="Name" 
+                               Stroke="#0450C2"
+                               ContainerType="Outlined"
+                               ContainerBackgroundColor="#E6EEF9">`
+    <Entry  />
+</inputLayout:SfTextInputLayout>  
+ 
+{% endhighlight %}
+
+{% highlight C# %} 
+
+var inputLayout = new SfTextInputLayout();
+inputLayout.Hint = "Name";
+inputLayout.ContainerType = ContainerType.Outlined;
+inputLayout.Stroke = Color.FromHex("#0450C2");
+inputLayout.ContainerBackgroundColor = Color.FromHex("#E6EEF9");
+inputLayout.Content = new Entry(); 
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![outlined](images/StatesAndColors/OutlinedContainerBackground.png)
+
+
+## Customizing the text color of label
+
+You can customize the text color of the `hint` label, counter label, `helper` label and `error` label using the `TextColor` property of the `LabelStyle`.
```

Comment:
You can customize the text color of the `hint` label, counter label, `helper` label, and `error` label using the `TextColor` property of the `LabelStyle.`

Reply:
Addressed the changes